### ISSUE TITLE
`PT006` & `PT007`: False negative with multiple `@pytest.mark.parametrize()` decorators

### ISSUE THREAD

**Issue by ngnpope** [charliermarsh/ruff#2630]
```python
import pytest

@pytest.mark.parametrize("a,b,c", [(1, 2), (3, 4), (5, 6)])  # Raises PT006
def test_single_decorator_for_pt006(a, b, c):
    pass

@pytest.mark.parametrize(("a", "b", "c"), ((1, 2), (3, 4), (5, 6)))  # Raises PT007
def test_single_decorator_for_pt007(a, b, c):
    pass

@pytest.mark.parametrize("a", [1, 2])  # OK
@pytest.mark.parametrize("b,c", [(3, 4), (5, 6)])  # Should raise PT006
def test_multiple_decorators_for_pt006(a, b, c):
    pass

@pytest.mark.parametrize("a", [1, 2])  # OK
@pytest.mark.parametrize(("b", "c"), ((3, 4), (5, 6)))  # Should raise PT007
def test_multiple_decorators_for_pt007(a, b, c):
    pass
```

Gives the following:

```console
❯ ruff --version
ruff 0.0.243

❯ ruff --isolated --select PT bug.py 
bug.py:3:26: PT006 [*] Wrong name(s) type in `@pytest.mark.parametrize`, expected `tuple`
bug.py:7:43: PT007 Wrong values type in `@pytest.mark.parametrize` expected `list` of `tuple`
Found 2 errors.
[*] 1 potentially fixable with the --fix option.
```

~~Makes me wonder whether there ought to be a new rule too to recommend using a single decorator?~~ _(Only makes sense if the arguments are the same length.)_

**Comment by ssbarnea** [comment]
I see a conflict here as ruff asks to use a list while flake8 asks for 
```
ruff:
PT007 Wrong values type in `@pytest.mark.parametrize` expected `list` of `tuple`

flake8:
PT007 wrong values type in @pytest.mark.parametrize, expected tuple of tuples
```

Basically there is no way to sort this for both ruff and flake8

**Comment by cidrblock** [comment]
A new entry in the pyproject.toml worked for me:

```
[tool.ruff.flake8-pytest-style]
parametrize-values-type = "tuple"
```

**Comment by charliermarsh** [comment]
IIUC both plugins allow this to be configured. "`list` of `tuple`" for both Ruff and flake8-pytest-style. Is it possible that you have flake8-pytest-style configured to enforce tuples of tuples?
